docs/reference/eql/functions.asciidoc

```diff
@@ -416,6 +417,71 @@ field datatypes:
 *Returns:* integer or `null`
 ====
 
+[discrete]
+[[eql-fn-match]]
+=== `match`
+
+Returns `true` if a source string matches one or more provided regular
```

Comment:
Do we know which regular expression is checked first? I mean, does it matter if it's matching the second or the fourth regular expression?

Reply:
I'm not sure it matters from a usage point of view.

The expressions are checked using `OR` logic so if _any_ of the regular expressions match, the function returns `true`.

The function also doesn't give any indicator of which expression matched. Only that at least one did.

`match("explorer.exe", "[a-z]*?.exe", "[1-9]")`
is equivalent to 
`match("explorer.exe", "[1-9]", "[a-z]*?.exe",)`